symfony/translation/3.3/manifest.json

```diff
@@ -1,3 +1,10 @@
 {
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%/",
+        "translations/": "translations/"
+    },
+    "container": {
```

Comment:
I know, it was on purpose as I wanted to remove this feature as I didn't had any usage anymore (that's also why it was not documented).